pkgs/by-name/fr/frescobaldi/package.nix

```
@@ -8,35 +8,40 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "frescobaldi";
-  version = "3.3.0";
-  format = "setuptools";
+  version = "4.0.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
-    owner = "wbsoft";
+    owner = "frescobaldi";
     repo = "frescobaldi";
     tag = "v${version}";
-    sha256 = "sha256-Q6ruthNcpjLlYydUetkuTECiCIzu055bw40O8BPGq/A=";
+    sha256 = "sha256-fTTHhoQJUOYncYkKb9jwp9i0hCoQpClvVlil/A6r8UI=";
   };
 
   propagatedBuildInputs = with python3Packages; [
```

Since it's moving to pyproject, consider this naming
```suggestion
  dependencies = with python3Packages; [
```
(see the Nixpkgs manual)